service-stubs/identity/org.wso2.carbon.identity.workflow.mgt.bps.stub/src/main/resources/ws-humantask-types.xsd

```diff
@@ -1,7 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-        <!--
-          Copyright (c) OASIS Open 2009. All Rights Reserved.
-        -->
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
 <xsd:schema
```

Comment:
you have removed 
 <!--
          Copyright (c) OASIS Open 2009. All Rights Reserved.
->
AFAIK we should keep that license here and we should not include WSO2 license since this is a copied file.